torchgeo/datasets/vhr10.py

```diff
@@ -0,0 +1,544 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Can we temporarily undo the filename change and move it to a separate PR (either before or after this PR is merged)? That way it will be easier to see in this PR which things actually changed and which didn't. Or did the majority of things change so the diff isn't actually useful?

Reply:
Rename reverted